File: Chavah.NetCore/Models/SignInStatus.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace BitShuva.Chavah.Models
{
    public enum SignInStatus
    {
        Success = 0,
        LockedOut = 1,
        RequiresVerification = 2,
        Failure = 3
    }
}

File: Chavah.NetCore/App/Models/SignInStatus.js
var BitShuva;
(function (BitShuva) {
    var Chavah;
    (function (Chavah) {
        var SignInStatus;
        (function (SignInStatus) {
            SignInStatus[SignInStatus["Success"] = 0] = "Success";
            SignInStatus[SignInStatus["LockedOut"] = 1] = "LockedOut";
            SignInStatus[SignInStatus["RequiresVerification"] = 2] = "RequiresVerification";
            SignInStatus[SignInStatus["Failure"] = 3] = "Failure";
        })(SignInStatus = Chavah.SignInStatus || (Chavah.SignInStatus = {}));
    })(Chavah = BitShuva.Chavah || (BitShuva.Chavah = {}));
})(BitShuva || (BitShuva = {}));
